docs/operate/get-started/supported-hardware/_index.md

```diff
@@ -117,7 +56,22 @@ After [installing `viam-server` or `viam-micro-server`](/operate/get-started/set
 
    Note that for microcontrollers, in order to add a module successfully to the machine’s configuration, the module needs to exist in the [firmware build](/operate/get-started/other-hardware/micro-module/).
 
-### Configure virtual hardware components
+## Browse supported hardware
```

Comment:
I'm really starting to see why this may not be needed here if we link to the Registry page in app above. (But sounds good to leave it here for now)

Reply:
Especially if the registry page had the built-ins :)